code/game/turfs/open/floor/misc_floor.dm

```diff
@@ -182,6 +182,7 @@
 	name = "rusted plating"
 	desc = "Corrupted steel."
 	icon_state = "plating_rust"
+	baseturfs = /turf/open/floor/plating/rust // VOID EDIT
```

Comment:
For what this infinite plating? Or you planning make planet entirely covered by plating?

Reply:
baseturfs are used for when a turf is destroyed. this will spawn another rust turf underneath it